letter-shop.md

```diff
@@ -0,0 +1,255 @@
+# Letter Shop Tutorial
+
+## What you need before you start:
+
+* a laptop with an internet connection and NodeJS installed
+* basic knowledge of the command line terminal
+* basic knowledge of JavaScript, including Promises and Buffers
+* (optional) the concept of one-way hash functions
```

Comment:
Also git if you're going to have people clone the repo

Reply:
So far I just added the scripts and a package.json, but the text still describe cut-and-pasting the files one-by-one.

Reply:
If you have the code there it seems better to just ask people to clone the repo instead of copying and pasting

Reply:
Seems like git is now a prerequisite